postgres/src/test/scala/zio/sql/postgresql/DeleteBatchSpec.scala

```diff
@@ -0,0 +1,36 @@
+package zio.sql.postgresql
+
+import zio.Cause
+import zio.test.Assertion._
+import zio.test._
+
+object DeleteBatchSpec extends PostgresRunnableSpec with DbSchema {
```

Comment:
Could you please add test where you would execute multiple queries and then sum the resulting number of changed lines? Also one test for insert and update would be nice.

Reply:
done